tests/files/test_model.xml

```diff
@@ -894,6 +895,37 @@
 		</Stream>
 
 	</Field>
+	<Field name="test_TransmissionCompressor">
+		<Process class="Before"/>
+		<Process class="TransmissionCompressor"/>
+		<Process class="Boundary" boundary="Production"/>
+		<Stream src="Before" dst="TransmissionCompressor">
+			<Contains>gas</Contains>
+			<Component name="CO2" phase="gas">119.35794</Component>
+			<Component name="H2" phase="gas">18.04195</Component>
+			<Component name="H2S" phase="gas">18.48612</Component>
+			<Component name="C1" phase="gas">108.77169</Component>
+			<Component name="C2" phase="gas">122.32514</Component>
+			<Component name="C3" phase="gas">119.59151</Component>
+			<Component name="C4" phase="gas">78.81646</Component>
+			<A name="temperature">60</A>
+			<A name="pressure">14.676</A>
```

Comment:
Note that these values come from peeking at the input stream contents when running the `gas_lifting_field` example.